selfie.c

```diff
@@ -4495,6 +4494,18 @@ void compile_cstar() {
 // ------------------------ MACHINE CODE LIBRARY -------------------
 // -----------------------------------------------------------------
 
+void emitRoundUp(uint64_t reg, uint64_t m) {
+  talloc();
+
+  // computes value(reg) -  ((value(reg) + (m - 1)) % m)
```

Comment:
I am not sure I understand the code. Does this really round up values?

Reply:
Oh my gosh, you are right. The comment is wrong. I fixed the comment in 3186adf4b088e04129337a7eda054b6e207488b2

First i was irritated that much, that I tried to prove the correctness of this formula. Which of course, is not possible. The good thing is, that I proved now also the correct version, after fixing the comment. 😃